src/components/activity/screens/ActivityAddEdit/views/ActivityAddEditView.js

```diff
@@ -49,8 +49,9 @@ export default class ActivityAddView extends React.Component {
   render() {
     return (
       <KeyboardAvoidingView
-        behavior="padding"
         style={{ flex: 1 }}
+        behavior=  { Platform.OS === 'ios'  ||  Platform.Version < 21 ? 'padding' : 'undefined'}
+        keyboardVerticalOffset= { Platform.OS === 'ios'  ||  Platform.Version > 20 ? 0 : StatusBar.currentHeight }
```

Comment:
According to the [react native](https://facebook.github.io/react-native/docs/platform-specific-code.html#platform-module) docs, Platform.Version behaves differently depending on Android or iOS, so we probably should ensure this is for Android we check the version (I assume the issue is for version below Android 20 ?). The condition might be unclear for Version = 20 between the two lines.

Reply:
I also have an error testing that : `Warning: Failed prop type: Invalid prop `behavior` of value `undefined` supplied to 'KeyboardAvoidingView', expected one of ["height","position","padding"].` I think the `'undefined'` should be just `undefined`